sdk/healthbot/azure-resourcemanager-healthbot/CHANGELOG.md

```diff
@@ -5,4 +5,4 @@
 
 ## 1.0.0-beta.1 (2021-01-07)
 
-- Azure Resource Manager Healthbot client library for Java. This package contains Microsoft Azure SDK for Healthbot Management SDK. Microsoft Healthcare Bot is a cloud platform that empowers developers in Healthcare organizations to build and deploy their compliant, AI-powered virtual health assistants and health bots, that help them improve processes and reduce costs. Package tag package-2020-12-08. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
\ No newline at end of file
+- Azure Resource Manager Healthbot client library for Java. This package contains Microsoft Azure SDK for Healthbot Management SDK. Microsoft Healthcare Bot is a cloud platform that empowers developers in Healthcare organizations to build and deploy their compliant, AI-powered virtual health assistants and health bots, that help them improve processes and reduce costs. Package tag package-2020-12-08. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
```

Comment:
I could not find what changed on the line (BTW this is generated from description in swagger spec, so if there is something wrong please let me know so I can fix the source there). But still OK to resourcemanager part.

Reply:
It is just add a "\n" at the end